src/components/Cohorts/CohortCard/style.ts

```diff
@@ -0,0 +1,84 @@
+import { styled, Button } from '@mui/material';
+
+const CohortCardDiv = styled('div')`
+  background-color: #fff;
+  border-radius: 0.3rem;
+  border: 1px solid transparent;
+  box-shadow: 0 0 15px 0 rgba(82, 63, 105, 0.05);
+  transition: 0.3s;
+  border-bottom: 3px solid transparent;
+  &:hover {
+    transform: translateY(-5px);
+    border-bottom-color: #0abb87;
+  }
+`;
+
+const CohortCardImgDiv = styled('div')`
+  width: 100%;
+  height: 150px;
+  background: url('../../../public/cohort-background.jpg') center center;
```

Comment:
When you have any file in the public folder you can access it with an absolute  path like this (`/public/file.jpg`).

One more thing if you can change the image to any image from `GSG` will be more beautiful. I will leave to you edited style for this component with adding glass effect

```tsx
const CohortCardImgDiv = styled('div')`
  width: 100%;
  height: 150px;
  background: url('https://mlwhex1ecbob.i.optimole.com/w:1814/h:1102/q:mauto/f:avif/https://gazaskygeeks.com/wp-content/uploads/2020/08/Copy-of-GAT1038.jpg') center center;
  background-repeat: no-repeat;
  background-size: cover;
  border-top-right-radius: 0.3rem;
  border-top-left-radius: 0.3rem;
  display: flex;
  justify-content: center;
  align-items: center;
  cursor: pointer;
  position: relative;

  &::after {
    content: '';
    position: absolute;
    width: 100%;
    height: 100%;
    background-color: #4a4a4a8c;
    backdrop-filter: blur(5px);
    border-top-right-radius: 0.3rem;
    border-top-left-radius: 0.3rem;
  }
`;
```
But Coz I used position absolute and relative you should edit the avatar style with adding `z-index: 1`.

```tsx
<Avatar
  alt={alt}
  src={imgSrc}
  sx={{
  width: 90, height: 90, boxShadow: '0 0 0 5px #ffffff', zIndex: 1,
  }}
/>
```